docs/source/includes/setup.rst

```diff
@@ -7,4 +7,5 @@ Setup
 
    getting_started.md
    development.md
-   setup_frontend_interface.md
+   running_choras.md
+   choras_coupling.md
```

Comment:
I feel like the sections 
- development.md 
- choras_coupling.md
would actually fit better under separate section called "developer guidelines", so moving them into a separate rst file or under a separate toctree would help get a better overview here I think.